### PR TITLE
Bookstore: Styling of the App.

### PR DESCRIPTION
**Pull request by mailsg**
Hi,

This PR showcases the styling of the `React-Redux` Bookstore App. 
The components are styled replicating the design in the `Zeplin` Project.

Kindly review the PR and provide feedback on the same.

Thanks.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,288 @@
...
+  align-items: center;
+  width: 100%;
+}
+
+.book-section-one {
+  width: 90%;
+  background-color: #fff;
+  height: 11rem;
+  margin-left: 3%;
+  padding: 20px;
```
Comment (brainconnect93) [inline src/App.css]: - [ ] Great effort working on the project styling, But currently there are somethings you need to do so as to have a perfect replica of the origin design. Below is a video recording of some places where your attention is needed in the project. I would kindly suggest you work on them and be sure you have a perfect design before submitting for review.

Video Claim 👇 

[screen-capture.webm](https://github.com/mailsg/Bookstore/assets/78634841/59b32868-e6e8-4f7d-9ef2-a9495148007e)

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
### Hi @mailsg, 👋 

# STATUS: Changes Required :recycle:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is a good use of Git flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (mailsg) [reply src/App.css]: @brainconnect93 the video is not working. Kindly check and let me know where do I need to make changes. 
image

**Review by alexander16108 — APPROVED** [review]
Hi @mailsg , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Flow
✅ Descriptive Pull Request

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**